linker/Mono.Linker.Steps/MarkStep.cs

```diff
@@ -28,21 +28,21 @@
 //
 
 using System;
-using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text.RegularExpressions;
 
 using Mono.Cecil;
 using Mono.Cecil.Cil;
+using Mono.Collections.Generic;
```

Comment:
This file now has both 
`using System.Collections.Generic;`
and
`using Mono.Collections.Generic;`
Is that intentional?